.rustfmt.toml

```diff
@@ -1,4 +1,15 @@
 version = "Two"
+edition = "2018"
 unstable_features = true
 error_on_line_overflow = true
 error_on_unformatted = true
+max_width = 110
```

Comment:
Why 110?

Reply:
same argument as above, it is reasonable, not too long or short, and made this codebase look the best in the experiments I ran.